lib/alert.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+import pytz
+
+from lib.alert_date import AlertDate
+
+
+class Alert:
+    def __init__(self, dict_):
+        for key, value in dict_.items():
+            setattr(self, key, value)
+
+        self.sent = AlertDate(self.sent)
+        self.expires = AlertDate(self.expires)
```

Comment:
You could use `SerialisedModel` from utils to keep this consistent with our other repos:

```suggestion
class Alert(SerialisedModel):
    ALLOWED_PROPERTIES = {
        'identifier',
        'message_type',
        'sent',
        'expires',
        'headline',
        'description',
        'static_map_png',
        'area_names',
    }

    @property
    def sent_date(self):
        return AlertDate(self.sent)

    @property
    def sent_date(self):    
        return AlertDate(self.expires)
```

Reply:
The main advantage of this is that it guards against accidentally leaving out properties in the YAML. Jinja is very happy to fail silently if a property is missing.

Reply:
Oh nice! I didn't realise we had the utils repo already. Having a separate property name for the converted dates also helps prevent confusion between the two types.

I've added a commit to convert to what you suggested 👍.